src/shared/models/setting/setting.service.ts

```diff
@@ -37,6 +39,19 @@ export class SettingService {
     await this.setObj<CustomSignUpFeesDto[]>('customSignUpFees', customSignUpFeesArray);
   }
 
+  async updateProcess(dto: UpdateProcessDto): Promise<void> {
+    const disabledProcesses = await this.getDisabledProcesses();
+    const index = disabledProcesses?.indexOf(dto.process);
+
+    index?.toString() && index !== -1 ? disabledProcesses.splice(index, 1) : disabledProcesses?.push(dto.process);
+
+    await this.setObj<Process[]>('disabledProcess', disabledProcesses ?? [dto.process]);
+  }
+
+  async getDisabledProcesses(): Promise<Process[]> {
+    return this.getObj<Process[]>('disabledProcess');
+  }
```

Comment:
Use with default value: `this.getObj<Process[]>('disabledProcess', []); `
And you can remove all the null checks in `updateProcess`